README.md

```diff
@@ -1,602 +1,361 @@
-# rspec-rails [![Build Status](https://secure.travis-ci.org/rspec/rspec-rails.svg?branch=master)](http://travis-ci.org/rspec/rspec-rails) [![Code Climate](https://img.shields.io/codeclimate/github/rspec/rspec-rails.svg)](https://codeclimate.com/github/rspec/rspec-rails)
-**rspec-rails** is a testing framework for Rails 3.x, 4.x and 5.x.
+# rspec-rails [![Build Status][]][travis-ci] [![Code Climate][]][code-climate]
 
-Use **[rspec-rails 1.x](http://github.com/dchelimsky/rspec-rails)** for Rails
-2.x.
+`rspec-rails` brings the [RSpec][] testing framework to [Ruby on Rails][]
+as a drop-in alternative to its default testing framework, Minitest.
 
-## Installation
+In RSpec, tests are not just scripts that verify your application code;
+they’re also detailed explanations of how the application is supposed to behave,
+expressed in plain English—in other words, _specs._
 
-Add `rspec-rails` to **both** the `:development` and `:test` groups in the
-`Gemfile`:
+Use **[`rspec-rails` 1.x][]** for Rails 2.x.
 
-```ruby
-group :development, :test do
-  gem 'rspec-rails', '~> 3.8'
-end
-```
+[Build Status]: https://secure.travis-ci.org/rspec/rspec-rails.svg?branch=master
+[travis-ci]: http://travis-ci.org/rspec/rspec-rails
+[Code Climate]: https://img.shields.io/codeclimate/github/rspec/rspec-rails.svg
+[code-climate]: https://codeclimate.com/github/rspec/rspec-rails
+[RSpec]: http://rspec.info/
+[Ruby on Rails]: https://rubyonrails.org/
+[`rspec-rails` 1.x]: http://github.com/dchelimsky/rspec-rails
 
-Want to run against the `master` branch? You'll need to include the dependent
-RSpec repos as well. Add the following to your `Gemfile`:
+## Installation
 
-```ruby
-%w[rspec-core rspec-expectations rspec-mocks rspec-rails rspec-support].each do |lib|
-  gem lib, :git => "https://github.com/rspec/#{lib}.git", :branch => 'master'
-end
-```
+1. Add `rspec-rails` to **both** the `:development` and `:test` groups
```

Comment:
Technically only the `:test` environment is required, but `:development` allows use of the generators, is it worth explaining that?